src/page-routes/PageRoutes.js

```diff
@@ -20,6 +20,10 @@ export default function PageRoutes() {
     window.scrollTo(0, 0);
   }, [location.pathname]);
 
+  useEffect(() => {
+    window.analytics.page(location.pathname);
+  }, [location]);
```

Comment:
I don't know how stable the reference is to the `location` object? might be better to have `location.pathname` be the dependency so that this will only fire when we have actually detected a change in path. (but if you tested this and it worked as you expected then this is probably fine too)

Reply:
Doesn't appear to have any functional impact but I agree with that choice regardless. Updated!